tailwind.config.cjs

```diff
@@ -56,7 +56,7 @@ module.exports = {
     colors: {
       // import colors from the color palette
       ...colorPalette,
-      
+
```

Comment:
```suggestion
      
```

Reply:
(shouldn't have this file being changed)

Reply:
Yeah the linter was harassing me, since this is the way the file should look after lint-staged I had to revert and commit manually :(